### DISCUSSION TITLE
connect to flutterflow

### DISCUSSION THREAD

**Discussion by macMikey** [serverpod/serverpod#3060] [General]
has anyone tried connecting a serverpod setup to a flutterflow project?
i'm thinking about using flutterflow to help make UI generation simpler, but hooking into those projects looks like it can be a little (but not too) tricky.

**Comment by tricaricom** [comment]
Agreed! After seeing the latest DreamFlow from FlutterFlow, I also think Serverpod integration is becoming more advantageous!
